### ISSUE TITLE
how to disable text input

### ISSUE THREAD

**Issue by anukool42** [toystars/react-native-multiple-select#123]
ow can i disable textinput(search)? i tried searchInputStyle={{ display: "none" }} and it does hide the text input but when i click on drop down the keyboard pops up. i want to disable it

**Comment by akiladevinda** [comment]
@anukool42  any fixes ?

**Comment by marcosperez** [comment]
try with `textInputProps={{ editable: false }}`

**Comment by AugustoAleGon** [comment]
@anukool42 The comment works for you?

**Comment by NathanLBCooper** [comment]
I don't think you want to use display none here. It messes with positioning. Try this:

    textInputProps={{ editable: false }}
    searchInputPlaceholderText=""
    searchIcon={false}

It kills the keyboard, 'hides' the text and hides the search icon.

**Comment by hetoms** [comment]
> I don't think you want to use display none here. It messes with positioning. Try this:
> 
> ```
> textInputProps={{ editable: false }}
> searchInputPlaceholderText=""
> searchIcon={false}
> ```
> 
> It kills the keyboard, 'hides' the text and hides the search icon.

For me the keyboard kept opening up, I had to add autoFocus: false

```
textInputProps={{ editable: false, autoFocus: false }}
```

**Comment by gregfenton** [comment]
@hetoms solution works for me (thanks!)

I recommend this issue be converted to a **NEW FEATURE**:
### Provide an option to avoid the search box being rendered entirely

**Comment by tanedward2101** [comment]
How can i disabled the entire thing? I want to lock it's value so it cannot be changed when meets certain condition.

**Comment by adk-echang** [comment]
Definitely hacky, and should be improved with a new disabled prop:
```
      items={disabled ? items.map(i => ({ ...i, disabled: true })) : items}
```
Disables all dropdown items, and you can change the submitButtonText to "Disabled" instead
